zetaclient/chains/bitcoin/observer/observer.go

```diff
@@ -635,6 +692,7 @@ func GetTxResultByHash(
 }
 
 // GetBlockHeightByHash gets the block height by block hash
+// TODO(revamp): move to client package
```

Comment:
same rpc.go ?

Reply:
As said, name can be rediscussed, these comments are for tracking purposes

Reply:
understood